pkg/utils/config_test.go

```diff
@@ -250,7 +250,7 @@ func TestFetchListingAddresses(t *testing.T) {
 	)
 	os.Setenv(
 		"CRAWL_CIVIL_LISTING_GRAPHQL_URL",
-		"http://localhost:8888/query",
+		"http://localhost:8889/query",
```

Comment:
Changed this to 8889 because jupyter notebooks run on 8888 and I didn't want to close the jupyter notebook I had running 😛

Reply:
lol, that's fine.